src/borg/archiver.py

```diff
@@ -2067,14 +2067,21 @@ def do_break_lock(self, args, repository):
             the re module <https://docs.python.org/3/library/re.html>`_.
 
         Path prefix, selector `pp:`
-            This pattern style is useful to match whole sub-directories. The pattern
-            `pp:root/somedir` matches `root/somedir` and everything therein.
+            This pattern style is useful to match whole sub-directories without worrying
+            about escaping special characters. Otherwise, you can just use the default
+            `fm:` or `sh:` styles, which also match whole sub-directories. Unlike those
+            styles, a trailing slash here makes no difference. The pattern `pp:root/somedir`
+            (and `pp:root/somedir/`) matches `root/somedir` and everything therein.
+
+            When used with exclude-norecurse rules, this style is equivalent to `pf:`
+            (except not as efficient).
 
         Path full-match, selector `pf:`
             This pattern style is (only) useful to match full paths.
             This is kind of a pseudo pattern as it can not have any variable or
-            unspecified parts - the full path must be given.
-            `pf:root/file.ext` matches `root/file.txt` only.
+            unspecified parts - the full path must be given. Trailing slashes make no
+            difference. `pf:root/file.ext` (and `pp:root/file.ext/`) matches `root/file.txt`
+            only.
```

Comment:
I guess it does not need the last sentence with an example for what a trailing slash is, especially not a weird one that rather looks like a quirk.